pillar.example

```diff
@@ -4,9 +4,7 @@ vmbuilder:
     vgname: "hypervisor_vg"
     lvm:
       - vgname: "hypervisor_vg"
-        devices: 
-          - "/dev/sdb5"
-          - "/dev/sda"
+        devices: "/dev/sdb5,/dev/sda"
```

Comment:
Iggy, backward compatibility became broken, because old syntax won't work in new saltstack version. But you are right, it' s better join to string in formula and then split to array in state then break compatibility. So I've fixed it in pull request #2. Also I've changed default saltmaster as you suggested. Thank you!